reference/resources/hostendpoint.md

```diff
@@ -23,12 +23,7 @@ to the interface.
 
 **Default behavior of external traffic to/from host**
 
-If a host endpoint is added and network policy is not in place, the {{ site.prodname }} default is to deny traffic to/from that endpoint (except for traffic allowed by failsafe rules). For host endpoints, {{ site.prodname }} blocks traffic only to/from interfaces that it’s been explicitly told about in network policy. Traffic to/from other interfaces is ignored.
-
-> **Note**: Currently, for host endpoints with `interfaceName: *`, only [pre-DNAT
-> policy]({{ site.baseurl }}/reference/host-endpoints/pre-dnat) is
-> implemented.
-{: .alert .alert-info}
+If a host endpoint is added and network policies or profiles are not in place selecting that host endpoint, the {{ site.prodname }} default is to deny traffic to/from that endpoint (except for traffic allowed by failsafe rules). For host endpoints, {{ site.prodname }} blocks traffic only to/from interfaces that it’s been explicitly told about in network policy. Traffic to/from other interfaces is ignored.
```

Comment:
It could confuse people to say profiles select host endpoints.  Profiles don't select host endpoints; it's more a case of host endpoint specify profiles to use.

I'm not sure I grok what this means "interfaces that it’s been explicitly told about in network policy".  Network policy doesn't say anything about interfaces?  And does "*" mean explicitly told about or not?

I would replace the note with something that explains k8s auto-host endpoints specify a default allow profile, so the behave similarly to pod workload endpoints.

Reply:
Yeah you're right, some of that existing content just reads wrong. I've moved some of the detail from the howto to this section: 
ce4f51c